docs/kubernetes/KubernetesIntegration.md

```diff
@@ -27,36 +27,40 @@ The `calico-kubernetes` plugin integrates directly with the Kubernetes `kubelet`
 
 We recommend using the latest version of [calicoctl](https://github.com/projectcalico/calico-docker/releases) to install both `calico/node` and `calico-kubernetes` on each of your nodes.
 
-## Installing Calico on a Kubernetes Master
-The Kubernetes master does not run any pods itself, and so does not typically require the `calico-kubernetes` plugin.  It does require the `calico/node` docker container so that individual pod IPs can be learned over BGP.
+## Installing Calico Componenets
+#### 1. Install Calico
+Each Kubernetes node requires both the `calico/node` container as well as the `calico-kubernetes` plugin.  Kubernetes Masters do not need the `calico-kubernetes` plugin installed, but may do so if pods will be scheduled on the master.
```

Comment:
You don't have a command that explains how to install just `calico/node` on the master. I think users would immediately run the full command with the `--kubernetes` flag if you don't specify what to call for the master.  I'm not sure how much impact that would have, if any.